### DISCUSSION TITLE
Add threshold to FuseOptionKeyObject?

### DISCUSSION THREAD

**Discussion by FGasper** [krisk/Fuse#595] [General]
Hello,

How possible would it be to do something like this?
```
keys: [
        // ----------------------------------------
        // Exact matches:
        {
            name: "name",
            weight: 1000,
            threshold: 0,
        },
        {
            name: "searchText",
            weight: 900,
            threshold: 0,
        },

        // ----------------------------------------
        // Fuzzy matches:

        {
            name: "name",
            weight: 800,
        },
        {
            name: "searchText",
            weight: 700,
        },
        {
            name: "category",
            weight: 600,
        },
        {
            name: "description",
            weight: 500,
        },
]
```
In other words, I’d like to prioritize, for only specific fields, exact matches above fuzzy matches. (Maybe this is already possible? Not sure.)

Thank you!

**Comment by krisk** [comment]
Well, technically exact matches should already move to the top, as they have scores nearest to zero. But if you want even more control, you _could_ use [extended search](https://fusejs.io/examples.html#extended-search). Perhaps you may also want to pair it with [logical query operations](https://fusejs.io/api/query.html)
